src/rabbit_mirror_queue_coordinator.erl

```diff
@@ -365,7 +365,12 @@ handle_cast({gm_deaths, DeadGMPids},
                                                    DeadPids),
             {stop, shutdown, State};
         {error, not_found} ->
-            {stop, normal, State}
+            {stop, normal, State};
+        {error, {not_synced, _}} ->
+            rabbit_log:error("Mirror queue ~p in unexpected state after gm_deaths."
```

Comment:
This error message is too much inside baseball.

Reply:
Can you clarify?

Reply:
"gm_deaths" is an implementation detail that makes no sense to the user.